baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ConsistentLoggerName.java

```diff
@@ -48,16 +50,13 @@
 
     @Override
     public Description matchVariable(VariableTree tree, VisitorState state) {
-        if (!matcher.matches(tree, state)) {
-            return Description.NO_MATCH;
+        if (matcher.matches(tree, state)) {
+            if (!tree.getName().contentEquals("log")) {
```

Comment:
pedantic nit: might as well combine conditions to reduce indent depth `if (matcher.matches(tree, state) && !tree.getName().contentEquals("log")) {`

I'll give this a try internally